hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/v2/components/tables/datanodesTable.tsx

```diff
@@ -0,0 +1,315 @@
+    render: (text: DatanodeState) => renderDatanodeState(text),
+    sorter: (a: Datanode, b: Datanode) => a.state.localeCompare(b.state)
+  },
+  {
+    title: 'Operational State',
+    dataIndex: 'opState',
+    key: 'opState',
+    filterMultiple: true,
+    filters: DatanodeOpStateList.map(state => ({ text: state, value: state })),
+    onFilter: (value, record: Datanode) => record.opState === value,
+    render: (text: DatanodeOpState) => renderDatanodeOpState(text),
+    sorter: (a: Datanode, b: Datanode) => a.opState.localeCompare(b.opState)
+  },
+  {
+    title: 'UUID',
+    dataIndex: 'uuid',
+    key: 'uuid',
+    sorter: (a: Datanode, b: Datanode) => a.uuid.localeCompare(b.uuid),
+    defaultSortOrder: 'ascend' as const,
+    render: (uuid: string, record: Datanode) => {
+      return (
+        //1. Compare Decommission Api's UUID with all UUID in table and show Decommission Summary
+        (decommissioningUuids && decommissioningUuids.includes(record.uuid) && record.opState !== 'DECOMMISSIONED') ?
+          <DecommissionSummary uuid={uuid} /> : <span>{uuid}</span>
+      );
+    }
+  },
+  {
+    title: 'Storage Capacity',
+    dataIndex: 'storageUsed',
+    key: 'storageUsed',
+    sorter: (a: Datanode, b: Datanode) => a.storageRemaining - b.storageRemaining,
+    render: (_: string, record: Datanode) => (
+      <StorageBar
+        strokeWidth={6}
+        capacity={record.storageTotal}
+        used={record.storageUsed}
+        remaining={record.storageRemaining}
+        committed={record.storageCommitted} />
+    )
+  },
+  {
+    title: 'Last Heartbeat',
+    dataIndex: 'lastHeartbeat',
+    key: 'lastHeartbeat',
+    sorter: (a: Datanode, b: Datanode) => moment(a.lastHeartbeat).unix() - moment(b.lastHeartbeat).unix(),
+    render: (heartbeat: number) => {
+      return heartbeat > 0 ? getTimeDiffFromTimestamp(heartbeat) : 'NA';
+    }
+  },
+  {
+    title: 'Pipeline ID(s)',
+    dataIndex: 'pipelines',
+    key: 'pipelines',
+    render: (pipelines: Pipeline[], record: Datanode) => {
+      const renderPipelineIds = (pipelineIds: Pipeline[]) => {
+        return pipelineIds?.map((pipeline: any, index: any) => (
+          <div key={index} className='pipeline-container-v2'>
+            <ReplicationIcon
+              replicationFactor={pipeline.replicationFactor}
+              replicationType={pipeline.replicationType}
+              leaderNode={pipeline.leaderNode}
+              isLeader={pipeline.leaderNode === record.hostname} />
+            {pipeline.pipelineID}
+          </div >
+        ))
+      }
+
+      return (
+        <Popover
+          content={
+            renderPipelineIds(pipelines)
+          }
+          title="Related Pipelines"
+          placement="bottomLeft"
+          trigger="hover">
+          <strong>{pipelines.length}</strong> pipelines
+        </Popover>
+      );
+    }
+  },
+  {
+    title: () => (
+      <span style={headerIconStyles} >
+        Leader Count
```

Comment:
In the old UI, we had a searchable Leader Count column, but in the new UI, this search functionality is missing. The same applies to the Container ID column.

<img width="471" alt="image" src="https://github.com/user-attachments/assets/a999b5ce-d212-4763-a3b7-3334bf12b155">

Reply:
I believe we can go over the searchable fields. For leader count it was just a number and I didn't think it was useful making it searchable